src/plugins/intel_gpu/src/plugin/ops/constant.cpp

```diff
@@ -97,8 +97,14 @@ static void create_data(ProgramBuilder& p, const ov::Shape& constDims, const std
         p.primitive_ids[initialconstPrimID] = constPrimID;
         p.profiling_ids.push_back(initialconstPrimID);
     } else {
-        GPU_DEBUG_LOG << "[" << initialconstPrimID << ": constant]" << std::endl;
+        if (constLayout.is_static() && constLayout.count() == 0) {
```

Comment:
I think constLayout should always static ?

Reply:
Fixed it. thanks